docs/core/index.md

```diff
@@ -68,3 +68,9 @@ Log out of the currently authenticated partner organization and store. The `logo
 $ shopify logout
 ```
 
+## `config analytics`
+
```

Comment:
Need a sentence just explaining `config` itself and its currently limited scope. Presumably we can expect more `config` options to be added in future.

```suggestion

Configure Shopify App CLI options. Currently there is one available option.

### `analytics`

```

Reply:
Hmm I'm worried "one available option" completely true, since the `help` command will return
`config: Change configuration of how the CLI operates
  Usage: shopify config [ feature | analytics ]
`
if devs have debug enabled 😕

Reply:
Updated with the feature subcommand too